src/components/services-view/services-view.html

```diff
@@ -1,8 +1,17 @@
 
 <div class="services-view">
 
-  <overlay-panel show-panel="$ctrl.orderingPanelvisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
-    <order-service service-class="$ctrl.serviceToOrder" handle-close="$ctrl.closeOrderingPanel"></order-service>
+  <overlay-panel show-panel="$ctrl.orderingPanelVisible" show-close="true" handle-close="$ctrl.closeOrderingPanel">
+    <order-service
+        ng-if="$ctrl.selectedServiceClass"
+        service-class="$ctrl.selectedServiceClass"
+        handle-close="$ctrl.closeOrderingPanel">
+    </order-service>
+    <builder-dialog
```

Comment:
@jeff-phillips-18 @jwforres Any strong opinion on verb vs noun for component names? I'll go with whatever everyone decides.

Here are the current component names (including the builder-dialog from this PR)

```
src/components/builder-dialog/builder-dialog.component.ts
src/components/landing-page/landing-page.component.ts
src/components/order-service/order-service.component.ts
src/components/overlay-panel/overlay-panel.component.ts
src/components/projects-summary/projects-summary.component.ts
src/components/saas-list/saas-list.component.ts
src/components/services-view/services-view.component.ts
```